docs/Installation.md

```diff
@@ -97,7 +97,7 @@ refers to the `MAJOR.MINOR` version of `0.1`, receiving bugfix patches such as `
 
 In order to link the Splunk RUM information with the distributed traces in the APM, certain configuration changes need to be applied to the APM agents monitoring the backend of the application. The changes are required to enable sending the `server-timing` header from the back-end which is then used to stitch together the spans from the browser with the backend trace. 
 
-To enable this, you need to adjust the configuration in the service the browser connects to that is monitored by Splunk APM agents and set `SPLUNK_CONTEXT_SERVER_TIMING_ENABLED` environmental variable to `true`. See manuals for corresponding APM agents for more information.
+To enable this, you need to adjust the configuration in the service the browser connects to that is monitored by Splunk APM agents and set `SPLUNK_CONTEXT_SERVER_TIMING_ENABLED` or  environmental variable to `true` for all agents that do not automatically enable this. Currently only Java agent ships with the configuration option enabled by default.  See manuals for corresponding APM agents for more information.
```

Comment:
or.....

Reply:
node.js also is enabled by default https://github.com/signalfx/splunk-otel-js#configuration-options-

Reply:
@ivomagi how about we provide something llike https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md in gdi-spec?

Reply:
Regarding compliance - we have a new writer in Instrumentals who started yesterday (Fabrizzio) - this is one of the three asks I have in store for him